instrumentation/spring-webflux-5.0/src/main/java/io/opentelemetry/auto/instrumentation/spring/webflux/server/HandlerAdapterAdvice.java

```diff
@@ -14,9 +14,9 @@
  * limitations under the License.
  */
 
-package io.opentelemetry.auto.instrumentation.springwebflux.server;
+package io.opentelemetry.auto.instrumentation.spring.webflux.server;
```

Comment:
our convention has been for the package name to be the module name minus the dashes and minus the version

i'm good with having a special case for spring (making this change), since we have many different spring modules (and growing), and this does look nicer

can you submit a follow-up PR to change this in the other spring auto modules so that we adopt this new spring module naming consistently?

Reply:
I'll make this my next change